docs/global/modules/core/pages/howto/modelql.adoc

```diff
@@ -0,0 +1,190 @@
+= ModelQL
+
+== Independent ModelQLClient
+
+ModelQL defines its own HTTP endpoint and provides server/client implementations for it.
+The `model-server` and the `mps-model-server-plugin` already implement this endpoint.
+The client can be created like this:
+
+[source,kotlin]
+--
+val client = ModelQLClient.builder().url("http://localhost/query").httpClient(httpClient).build()
```

Comment:
What is that V2 client being used here: https://github.com/modelix/modelix.core/pull/295/files#diff-51d9193db8d0f6113245adf5998e107ef545cab09c37b191dfd0e8c9bdb81296R95 Is that what needs to be used in the future?